core/state-machine/src/lib.rs

```diff
@@ -291,7 +291,7 @@ where
 		.to_vec();
 
 	let heap_pages = try_read_overlay_value(overlay, backend, well_known_keys::HEAP_PAGES)?
-		.and_then(|v| u64::decode(&mut &v[..])).unwrap_or(8) as usize;
+		.and_then(|v| u64::decode(&mut &v[..])).unwrap_or(1024) as usize;
```

Comment:
Could we add a constant for this (1024)? Somewhere up, there is the same change and no one does probably know the meaning of this.